rpc/block_svc_routines.c

```diff
@@ -626,6 +626,7 @@ glusterBlockCallRPC_1(char *host, void *cobj,
   }
 
  out:
+  ret = reply.exit;
```

Comment:
This should be moved prior to label.
Also, make sure failed exists should reflect 'ret = -1;' as I see some reset of 'ret' set at initialization.

Reply:
ok.